example/stop-resume/index.html

```diff
@@ -0,0 +1,55 @@
+<head>
+  <style> body { margin: 0; } </style>
+
+  <script src="//unpkg.com/3d-force-graph"></script>
+  <!-- <script src="../../dist/3d-force-graph.js"></script> -->
+</head>
+
+<body>
+  <button id="stop-resume" style="display: block; margin: 20px; height: 30px; width: 100px;">Stop/Resume</button>
+  <div id="3d-graph"></div>
+
+  <script>
+    // Random tree
+    const N = 300;
+    const gData = {
+      nodes: [...Array(N).keys()].map(i => ({ id: i })),
+      links: [...Array(N).keys()]
+        .filter(id => id)
+        .map(id => ({
+          source: id,
+          target: Math.round(Math.random() * (id-1))
+        }))
+    };
+
+    const distance = 1000;
+
+    const Graph = ForceGraph3D()
+      (document.getElementById('3d-graph'))
+        .enableNodeDrag(false)
+        .enableNavigationControls(false)
+        .showNavInfo(false)
+        .cameraPosition({ z: distance })
+        .graphData(gData);
+
+    // camera orbit
+    let angle = 0;
```

Comment:
Adding the camera orbit in this example may be a little confusing. It's also cancelling the Navigation controls and Node drag, which is important to outline that they will not function when the rendering cycle is paused.